.github/workflows/loadtest.yaml

```diff
@@ -0,0 +1,45 @@
+name: Performance Test
+on: 
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'K6 Load Test'  
+
+jobs:
+  k6_test_run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          repository: longwuyuan/ingress-nginx
```

Comment:
Change this so we can use in prod / merge pls :)

Reply:
sorry. was a typo during copy/paste. fixed now. Please check